ies-route53/recipes/add.rb

```diff
@@ -1,14 +1,8 @@
 instance = get_instance
 
-host_name = get_hostname(node, true)
-stack_name = get_normalized_cluster_name(node)
-region_id = instance['region']
-public_ip = instance['ip']
-record_name = "#{host_name}.#{stack_name}.#{region_id}"
-
 ies_route53_record 'create a record' do
-  name                  record_name
-  value                 public_ip
+  name                  get_record_name
```

Comment:
@fh Yes, take a look at this commit:
https://github.com/till/easybib-cookbooks/commit/ea505d1853d7b100ea45637f98aee1aff4cfe855

(I needed the hostname for papertrail as well, so I refactored this into a helper and re-used it.)